scripts/run_lighthouse_tests.py

```diff
@@ -196,7 +196,8 @@ def run_lighthouse_checks(lighthouse_mode: str) -> None:
     bash_command = [
         common.NODE_BIN_PATH, lhci_path, 'autorun',
         '--config=%s' % LIGHTHOUSE_CONFIG_FILENAMES[lighthouse_mode],
-        '--max-old-space-size=4096'
+        '--max-old-space-size=4096',
+        '--no-sandbox'
```

Comment:
Why do we need to disable the sandbox? We should document the reasoning behind this decision